ci/build-linux.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
What about `/bin/bash`?

`/bin/sh` means different things in different distros these days, e.g.,

* on Ubuntu, `/bin/sh` means `/bin/dash`, see https://wiki.ubuntu.com/DashAsBinSh
* same on Debian: https://wiki.debian.org/DashAsBinSh
* on CentOS `/bin/sh` is really `/bin/bash`: https://unix.stackexchange.com/questions/197488/centos-7-shell-script-interpretor
* presumably, RedHat is similar to CentOS

So, to be consistent and to have the same flags, etc. everywhere, please consider using `/bin/bash` on all scripts and platforms.

Reply:
/bin/bash is not a thing everywhere.   `/usr/bin/env bash` if we use any bash features, which we do not, so `/bin/sh` is Okay I think.